flaky/utils.py

```diff
@@ -2,6 +2,8 @@
 
 from __future__ import unicode_literals
 
+from six import binary_type
```

Comment:
I like `six`, but it doesn't make sense to add a new dependency just for this one use.

Right below this, we're manually determining the `unicode_type`. If the `try` succeeds, then we are in Python 2 and can set `binary_type` to `str`. And if it fails, then we are in Python 3 and can set `binary_type` to `bytes`.